yoti_python_sdk/profile.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, profile_attributes):
                 if field.name == config.ATTRIBUTE_STRUCTURED_POSTAL_ADDRESS:
                     self.try_convert_structured_postal_address_to_dict(field, anchors)
 
-            self.ensure_postal_address(anchors)
+            self.ensure_postal_address()

```

Comment:
Why is this changing?

Reply:
PyCharm alerted me that it's possible anchors could not be initialised before it gets here. I've changed the behaviour of `ensure_postal_address` to use the structured_postal_address anchors directly there instead